tests/test_authenticators.py

```diff
@@ -51,6 +57,8 @@ class TestAuthenticators(unittest.TestCase):
     def setUpClass(cls):
         cls.user = user = pwd.getpwuid(os.getuid())
         cls.group = group = grp.getgrgid(user.pw_gid)
+        if cls.user.pw_uid == 0 or cls.group.gr_gid == 0:
+            raise ValueError("Don't run tests as root!")
```

Comment:
I think we need to allow to run tests as any user, we can wanr maybe if that means the test will have unexpected behavior, but they shouldn't